react/src/components/molecules/CalloutLink/CalloutLink.stories.js

```diff
@@ -8,7 +8,7 @@ import CalloutLink from './index';
 import CalloutLinkDocs from './CalloutLink.md';
 import calloutLinkOptions from './CalloutLink.knobs.options';
 
-storiesOf('molecules/CalloutLink', module).addDecorator(withKnobs)
```

Comment:
I think these changes resulted in stories being listed in two places
![doubled-stories](https://user-images.githubusercontent.com/38226003/44478585-2972ae00-a60c-11e8-9259-168e81074989.png)

Reply:
Could we do what we discussed about using the `add.("[ComponentName]")` in stories to create the reference url?